src/Widgets/FormStepContainerWidget/components/SelectDropdownComponent.tsx

```diff
@@ -1,12 +1,17 @@
 import * as React from "react";
 import { DropdownOption } from "./DropdownOption";
-
-export const Dropdown = ({ options, name, id, required }) => {
+type props = {
+  options: string[];
+  name: string;
+  id: string;
+  required: boolean;
+};
+export const Dropdown = ({ options, name, id, required }: props) => {
```

Comment:
Here is example of how we define types for React compoenents in Neoletter GUI.

https://github.com/infopark/dxhub/blob/main/neoletter-gui/src/components/topics/buttons/DeleteTopicButton.tsx

Reply:
Did that for every React Component in the package.
Done